lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/model/MorphiaCollection.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *  
+ * Copyright (C) since 2016 the original author or authors.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.storage.mongodb.model;
+
+import io.lumeer.api.model.Attribute;
+import io.lumeer.api.model.Collection;
+import io.lumeer.storage.mongodb.model.common.MorphiaResource;
+import io.lumeer.storage.mongodb.model.embedded.MorphiaAttribute;
+
+import org.mongodb.morphia.annotations.Embedded;
+import org.mongodb.morphia.annotations.Entity;
+import org.mongodb.morphia.annotations.Field;
+import org.mongodb.morphia.annotations.Index;
+import org.mongodb.morphia.annotations.IndexOptions;
+import org.mongodb.morphia.annotations.Indexes;
+import org.mongodb.morphia.annotations.Property;
+import org.mongodb.morphia.utils.IndexType;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+@Indexes({
+      @Index(fields = { @Field(MorphiaCollection.CODE) }, options = @IndexOptions(unique = true)),
+      @Index(fields = { @Field(MorphiaCollection.NAME) }),
```

Comment:
Name should be also unique, or no?

Reply:
It depends. There is basically no special use of name on the back-end side which would require making it unique. However, it does make sense from the user's perspective since they only deal with the names and see codes just in the URL.